internal/tool/multi/deployer.go

```diff
@@ -109,6 +111,33 @@ type handler struct {
 
 var _ envelope.EnvelopeHandler = &handler{}
 
+// Ofcourse this will read and parse values from config in final
+// version of this PR.
+func createCgroupManager() (*cgroups.Manager, error) {
+	// Note: In final PR these parmeters will be read and
+	// parsed from the config.
+	group := "/service-weaver-cgroup"
+	groupPath := fmt.Sprintf("%s-%d", group, os.Getpid())
+	var (
+		quota  int64  = 5000
+		period uint64 = 100000
+		weight uint64 = 100
+	)
+	res := cgroups.Resources{
+		CPU: &cgroups.CPU{
+			Weight: &weight,
+			Max:    cgroups.NewCPUMax(&quota, &period),
+			Cpus:   "0",
+			Mems:   "0",
+		},
+	}
+	cgroupManager, err := cgroups.NewManager("/sys/fs/cgroup", groupPath, &res)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to create a new cgroup: %w", err)
+	}
+	return cgroupManager, nil
+}
+
```

Comment:
This is just a stub, if approved in final PR i will refactor this bit to show up from `config` files.